### DISCUSSION TITLE
How export all the bookmarks

### DISCUSSION THREAD

**Discussion by ja-Bertolin** [AlexandreRouma/SDRPlusPlus#590] [Q&A]
Hi

Is there any way to export all the bookmarks in one shot? and not one by one? I have created several groups with dozens of frequencies in each one and, in order to take a backup of them, I would like to export them to a unique file

Thanks

**Comment by AlexandreRouma** [comment]
If you hold the control key you can select multiple entries. Just select all the entires.

If you want to backup everything, copy the frequency_manager_config.json

Note for future readers: the way of storing bookmarks will change relatively soon, so disregard this if you're reading this a few months from now.
